rts/Game/CameraHandler.cpp

```diff
@@ -379,6 +569,7 @@ void CCameraHandler::SetCameraMode(unsigned int newMode)
 	CCameraController* newCamCtrl = camControllers[currCamCtrlNum = newMode];
 
 	newCamCtrl->SetPos(oldCamCtrl->SwitchFrom());
+	newCamCtrl->SetRot(oldCamCtrl->GetRot());
```

Comment:
See previous comment, is this a vector to pass a previously-impossible rotation by switching from a less constrained camera to a more constained one?